cli/src/commonMain/kotlin/OrtServerMain.kt

```diff
@@ -55,8 +55,8 @@ fun main(args: Array<String>) {
         }
 
         exitProcess(0)
-    } catch (e: AuthenticationException) {
-        cli.echoError(e.message)
+    } catch (@Suppress("SwallowedException") e: AuthenticationException) {
```

Comment:
BTW, I believe you could avoid those `Suppress` annotations by ruing `runCatching` / `onFailure` with a `when` clause inside.

Reply:
I added a commit with this refactoring.